frontend/const.ts

```diff
@@ -0,0 +1,157 @@
+import NeoIcon from "@/assets/svg/files/neo-icon.svg";
+import DecredIcon from "@/assets/svg/files/decred-icon.svg";
+import BinanceIcon from "@/assets/svg/files/binance-icon.svg";
+import LitecoinIcon from "@/assets/svg/files/litecoin-icon.svg";
+import BitcoinIcon from "@/assets/svg/files/bitcoin-icon.svg";
+import ckBtcIcon from "@/assets/svg/files/ckbtc.svg";
+import OpenChatIcon from "@/assets/svg/files/openchat.svg";
+import KinicIcon from "@/assets/svg/files/kinic.svg";
+import HotOrNotIcon from "@/assets/svg/files/hot-or-not.svg";
+import DragginzIcon from "@/assets/svg/files/dragginz.svg";
+import { z } from "zod";
+import { Token } from "@redux/models/TokenModels";
+
+// Enums
+export const TransactionTypeEnum = z.enum(["RECEIVE", "SEND", "NONE"]);
+export type TransactionType = z.infer<typeof TransactionTypeEnum>;
+
+export const OperationStatusEnum = z.enum(["COMPLETED", "REVERTED", "PENDING"]);
+export type OperationStatus = z.infer<typeof OperationStatusEnum>;
+
+export const OperationTypeEnum = z.enum(["TRANSACTION", "FEE"]);
+export type OperationType = z.infer<typeof OperationTypeEnum>;
+
+export const DrawerOptionEnum = z.enum(["SEND", "RECEIVE", "WRAP"]);
+export type DrawerOption = z.infer<typeof DrawerOptionEnum>;
+
+export const IconTypeEnum = z.enum(["ASSET", "HEADER", "FILTER"]);
+export type IconType = z.infer<typeof IconTypeEnum>;
+
+export const AssetSymbolEnum = z.enum(["ICP", "ckBTC", "CHAT", "KINIC", "SNS1", "HOT"]);
+export type AssetSymbol = z.infer<typeof AssetSymbolEnum>;
+
+export const ThemesEnum = z.enum(["dark", "light"]);
+export type Themes = z.infer<typeof ThemesEnum>;
+
+export const SendingStatusEnum = z.enum(["sending", "done", "error", "none"]);
+export type SendingStatus = z.infer<typeof SendingStatusEnum>;
+
+export const AddingAssetsEnum = z.enum(["adding", "done", "error", "none"]);
+export type AddingAssets = z.infer<typeof AddingAssetsEnum>;
+
+export const TokenNetworkEnum = z.enum(["ICRC-1"]);
+export type TokenNetwork = z.infer<typeof TokenNetworkEnum>;
+
+export const AccountDefaultEnum = z.enum(["Default"]);
+export type AccountDefault = z.infer<typeof AccountDefaultEnum>;
+
+export const WorkerTaskEnum = z.enum(["TRANSACTIONS", "ASSETS"]);
+export type WorkerTask = z.infer<typeof WorkerTaskEnum>;
+
+export const AuthNetworkNameEnum = z.enum(["Internet Identity", "NFID", "Metamask", "xxxx", "NONE"]);
+export type AuthNetworkName = z.infer<typeof AuthNetworkNameEnum>;
+
+export const AuthNetworkTypeEnum = z.enum(["IC", "NFID", "MM", "NONE"]);
+export type AuthNetworkType = z.infer<typeof AuthNetworkTypeEnum>;
+
+export const DeleteContactTypeEnum = z.enum(["CONTACT", "ASSET", "SUB"]);
+export type DeleteContactTypeEnum = z.infer<typeof DeleteContactTypeEnum>;
+
+export const TimerActionTypeEnum = z.enum(["TRANSACTIONS", "ASSETS"]);
+export type TimerActionType = z.infer<typeof TimerActionTypeEnum>;
+
+//
+export const ICRC1systemAssets: Array<Token> = [
+  {
+    id_number: 9999,
```

Comment:
Is this ok to have multiple with id_number 9999? Can it cause any problems?

Reply:
Just a initial number, then when a token is added a new id will be asigned. However I'll change a different id for each.